mava/wrappers/jumanji.py

```diff
@@ -169,102 +178,52 @@ def observation_spec(self) -> specs.Spec[Observation]:
         )
 
 
-class RwareMultiAgentWrapper(Wrapper):
-    """Multi-agent wrapper for the Robotic Warehouse environment."""
-
-    def reset(self, key: chex.PRNGKey) -> Tuple[State, TimeStep]:
-        """Reset the environment. Updates the step count."""
-        state, timestep = self._env.reset(key)
-        timestep.observation = Observation(
-            agents_view=timestep.observation.agents_view,
-            action_mask=timestep.observation.action_mask,
-            step_count=jnp.repeat(timestep.observation.step_count, self._env.num_agents),
-        )
-        return state, timestep
-
-    def step(self, state: State, action: chex.Array) -> Tuple[State, TimeStep]:
-        """Step the environment. Updates the step count."""
-        state, timestep = self._env.step(state, action)
-        timestep.observation = Observation(
-            agents_view=timestep.observation.agents_view,
-            action_mask=timestep.observation.action_mask,
-            step_count=jnp.repeat(timestep.observation.step_count, self._env.num_agents),
-        )
-        return state, timestep
-
-    def observation_spec(self) -> specs.Spec[Observation]:
-        """Specification of the observation of the `RobotWarehouse` environment."""
-        step_count = specs.BoundedArray(
-            (self._env.num_agents,),
-            jnp.int32,
-            [0] * self._env.num_agents,
-            [self._env.time_limit] * self._env.num_agents,
-            "step_count",
-        )
-        return self._env.observation_spec().replace(step_count=step_count)
-
-
-class RwareMultiAgentWithGlobalStateWrapper(Wrapper):
-    """Multi-agent wrapper for the Robotic Warehouse environment.
+class GlobalStateWrapper(Wrapper):
```

Comment:
I wonder if we shouldn't have a `wrapper/mava.py` file that includes the more general wrappers like the agent IDs and the global state wrappers? Since I foresee us reusing these for any env we wrap.

Reply:
I really like this idea 🔥 ! We can name the file mava.py (like jumanji.py, gym.py ...) and then we change and adjust the environments wrapper separately 🙌

Reply:
@sash-a, keen to hear your thoughts here? I think we should everything jumanji specific in this jumanji file and move general wrappers to another file. Calling the file `mava.py` is a bit misleading since Mava isn't an environment. My suggestion is to call it `shared.py` since it will be wrappers shared between envs.

Reply:
I think that is okay, since the base Mava API is an extension of the Jumanji API. Thank you for the changes :raised_hands: